yaml-tests/src/test/resources/bytes.yamsql

```diff
@@ -26,6 +26,7 @@ setup:
     - query: insert into lb values (1, x'deadbeef'), (2, b64'yv4='), (3, null);
 ---
 test_block:
+  name: bytes-tests
```

Comment:
please make the name optional, defaulting to the YAML file's name if not set explicitly.

Reply:
The name is optional. The current name if not given is `unnamed-` followed by an index that indicates the position of the block in the file.